Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm

```diff
@@ -121,9 +121,6 @@ static FragmentAndResources createFragment(Frame&, NSAttributedString *)
 
     NSURL *baseURL = URL::fakeURLWithRelativePart(emptyString());
 
-    // The output base URL needs +1 refcount to work around the fact that NSHTMLReader over-releases it.
-    CFRetain((__bridge CFTypeRef)baseURL);
```

Comment:
This should be in a separate change since it has nothing to do with what you're mechanically fixing at the other two callsites, and is - unlike those - a behavior change that needs double checking and justification.

Reply:
I will deal with it

Reply:
Done!